pages/thanks-rider.html

```diff
@@ -30,4 +41,5 @@
             </div>
         </div>
     </div>
+    <script src="/scripts/thanks.js"></script>
```

Comment:
To place this in the footer.html partial with a conditional comment, you need to add a variable to the front-matter at the top of the file, e.g. like this:
```
---
layout: default
permalink: thanks-driver/
title: Thank you!
thanks: true
---
```

Then you can add the script in footer.html like this:
```
{% if page.thanks %}
    <script src="/scripts/thanks.js"></script>
{% endif %}
```